packages/app/src/background/services/zkIdentity/index.ts

```diff
@@ -397,7 +208,7 @@ export default class ZkIdentityService extends BaseService implements IBackupabl
     return bigintToHex(identity.genIdentityCommitment());
   };
 
-  private insertIdentity = async (newIdentity: ZkIdentitySemaphore): Promise<boolean> => {
+  private insertIdentity = async (newIdentity: ZkIdentitySemaphore, urlOrigin?: string): Promise<boolean> => {
```

Comment:
Lets remove unneeded Identity from names, just `insert`.
```suggestion
  private insert = async (newIdentity: ZkIdentitySemaphore, urlOrigin?: string): Promise<boolean> => {
```

The same goes for `loadIdentities`,  `createIdentity`,  `createIdentityRequest`,  `deleteIdentity`,  `deleteAllIdentities`
```ts
  private load = async () => {
  
create = async ({
    walletType,
    messageSignature,
    isDeterministic,
    groups,
    urlOrigin,
    options,
  }: INewIdentityRequest): Promise<string | undefined> => {
  
  createRequest = async ({ urlOrigin }: ICreateIdentityRequestArgs): Promise<void> => {
    await this.browserController.openPopup({ params: { redirect: Paths.CREATE_IDENTITY, urlOrigin } });
  };
  
    delete = async ({ identityCommitment }: IDeleteIdentityArgs): Promise<IDeleteIdentityArgs> => {
    
     deleteAll = async (): Promise<boolean> => {
```

For this one: `writeIdentities` could be renamed into `save`

```ts
private save = async (identities: Map<string, string>): Promise<void> => {
    const serializedIdentities = JSON.stringify(Array.from(identities.entries()));
    const ciphertext = this.cryptoService.encrypt(serializedIdentities, { mode: ECryptMode.MNEMONIC });
    await this.identitiesStore.set(ciphertext);
  };
```

Reply:
It's not in a scope of this PR.

Reply:
Yeah, I just added an issue for that.